runtime/altair/src/lib.rs

```diff
@@ -1350,6 +1352,38 @@ impl pallet_membership::Config for Runtime {
 	type WeightInfo = pallet_membership::weights::SubstrateWeight<Self>;
 }
 
+parameter_types! {
+	#[derive(Clone, PartialEq, Eq, Debug, TypeInfo, Encode, Decode, MaxEncodedLen)]
+	pub const MaxFeedersPerKey: u32 = 10;
+	pub const FirstValueFee: Fee = Fee::Balance(10 * CFG); // TODO: Specify
```

Comment:
Reminder for this `TODO`. We should make use of the `deposit` function.

Reply:
Sorry, I thought we were utilizing this const function from Substrate already, but I was wrong: https://github.com/paritytech/substrate/blob/5e49f6e44820affccaf517fd22af564f4b495d40/bin/node/runtime/src/constants.rs#L28

For oracles, I suggest to set items to the appropriate `Max*` bound for the number of values per key and `bytes` as the max size of a value. Then, we only need to apply weights to both parameters. For that, we should be lower than Polkadot, maybe Kusama configuration would be a good inspiration.

Reply:
Mmm... OK, I see. That represents somehow the byte cost of storing things on the chain, right?

In our case, the fee is for 1 item of `bytes(OracleKey) + bytes(OracleValue)` bytes

Do you know if our codebase currently has any function to handle this, or would I need to create one?

Reply:
> Mmm... OK, I see. That represents somehow the byte cost of storing things on the chain, right?

Exactly. Anything else is arbitrary and could expose an attack vector if the deposit is too low for the occupied storage. Since oracles can be fed by anyone, we should use a similar function IMO. But it should not be a blocker for this PR and can be done in a subsequent one.

> Do you know if our codebase currently has any function to handle this, or would I need to create one?

Yes, we actually have and use it (at least for Proxy and Preimages) with the same configuration as above (with `CENTI_CFG` instead of `CENTS`): https://github.com/centrifuge/centrifuge-chain/blob/c4f05ca273ed14d38b484b2f3bdd5c0dd66dbb70/libs/primitives/src/lib.rs#L260-L262

So we could end up with 
```rust
	pub const FirstValueFee: Balance = deposit(1, 4 + 16)
```

which would be 135 `CENTI_CFG` or `1.35` CFG. To me, that appears to be a reasonable fee for occupying storage and protecting against spamming. In the end, all fees benefit our treasury and thus the longterm sustainability of the protocol 🙃

Reply:
Thanks for expanding this, @wischli; I agree with all you've said!

Reply:
Just one question. Does the bytes of the key represent the hash representation of such key in the map? (I think 32 bytes)

Reply:
It probably does, I am not entirely sure. I would just work with the bytes of the storage items: `OracleKey (u8)`, `OracleValue (u128)`, `Timestamp (u64)`. So I would adjust my above stated `135 CENTI_CFG` to `165 CENTI_CFG`.

For the `pallet_preimage`, one preimage byte costs exactly `deposit(0, 1)`.